frigate/config.py

```diff
@@ -530,6 +530,7 @@ class CameraUiConfig(FrigateBaseModel):
 
 class CameraConfig(FrigateBaseModel):
     name: Optional[str] = Field(title="Camera name.", regex="^[a-zA-Z0-9_-]+$")
+    enabled: Optional[bool] = Field(default=True, title="Enable camera.")
```

Comment:
This isn't optional, it always needs a value of true or false

Reply:
I tested the different options in the config (with two cameras A and B):
- no entry "enabled: ..." --> both cameras are shown and no error is thrown (I assumed this is a result of the optional Bool setting)
- entry "enabled: False" for Camera A or B --> Camera A or B is disabled
- entry "enabled: True" for Camera A or B --> Camera A or B is enabled.

I do not really understand what might be wrong here.

Reply:
You are using the `Optional` which means the value is nullable, but it shouldn't be. This will always have a default value of True or a user set value.